SPECS/cni/cni.spec

```diff
@@ -65,13 +66,13 @@ the container is deleted. Because of this focus, CNI has a wide
 range of support and the specification is simple to implement.
 
 %prep
-%setup -q
+%autosetup -N
 cp %{SOURCE2} build.sh
-
-%build
 # create vendor folder from the vendor tarball and set vendor mode
 tar -xf %{SOURCE3} --no-same-owner
+%patch 0 -p1 -d "vendor/golang.org/x/text"
```

Comment:
Since we're already moving thing around here I think we can drop the `tar -xf %{SOURCE3} --no-same-owner` and `%patch` lines completely and modify `%autosetup` instead to use this:
```
%autosetup -a 3 -p1
```

The `-a 3` argument tells `autosetup` to extract `Source3` in addition to `Source0`.